lib/reek/ast/sexp_extensions/send.rb

```diff
@@ -10,6 +10,10 @@ def receiver
           children.first
         end
 
+        def value_assigned
```

Comment:
Since this method does not makes sense for all `:send` nodes, you should move this logic to its caller.